### PR TITLE
Adds host scoping pseudo-element

### PR DESCRIPTION
**Pull request by vwallen**

### CODE REVIEW

File: features/host.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Host
+description: The `::host` CSS pseudo-element selects the containing element of the shadow tree in which it is used.
```
Comment (vwallen) [inline features/host.yml:2]: Unclear if the description should be extended to include `::host()` and `::host-context()` explicitly. I left them off for brevity

Comment (jamesnw) [reply features/host.yml:2]: I think `:host-context()` should be a separate feature, based  on support, and `:host()` should be mentioned in the description here.

Comment (jamesnw) [inline features/host.yml:2]:
```suggestion
description: The `:host` CSS pseudo-class selects the containing element of the shadow tree in which it is used.
```

File: features/host.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+spec: https://drafts.csswg.org/css-scoping-1/
```
Comment (jamesnw) [inline features/host.yml:3]:
```suggestion
https://drafts.csswg.org/css-scoping-1/#host-selector
```

**Review by jamesnw — CHANGES_REQUESTED** [review]

**Review by jamesnw — APPROVED** [review]

**Pull request merged**